features/guidance/turn-angles.feature

```diff
@@ -742,6 +742,7 @@ Feature: Simple Turns
 
     #http://www.openstreetmap.org/#map=19/52.53293/13.32956
     Scenario: Curved Exit from Curved Road
+        Given a grid size of 5 meters
         Given the node map
```

Comment:
@MoKob Can you approve this change as well? The issue I was seeing with a grid size of 1m as it was before is that it wasn't accurately representing the linked OSM intersection.

While debugging, I got around to logging the calculated angle deviation for each intersection, and somehow the angle for `dhij` was `19.9583`, and the angle for `defg` was `27.2692`, which I believe misrepresents the scenario we're trying to model. This explains why the minor `Siemens` street was always being considered the obvious turn.

As a result, when the 90 degree turn onto the residential `Siemens` street is properly modelled the instructions returned also needed to be updated as proper turns rather than slight turns.